application/controllers/Homepage.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Homepage extends Application
+{
+
+	/**
+	 * Index Page for this controller.
+	 */
+	public function index()
+	{
+		$this->data['pagebody'] = 'homepage';
+		$this->data['fleetsize'] = $this->fleet->size();
+		//$this->render();
+		$this->render();
```

Comment:
Duplicate line commented out in line above

Reply:
removed